core/src/com/nuclearthinking/myheroagency/controller/Assets.java

```diff
@@ -0,0 +1,45 @@
+package com.nuclearthinking.myheroagency.controller;
+
+import com.badlogic.gdx.assets.AssetManager;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+
+/**
+ * Created by Izonami on 10.05.2016.
+ */
+public class Assets {
+
+    private static Assets instance;
+    private static AssetManager assetManager;

```

Comment:
Caution: don't make your AssetManager or any other resources (like Texture, etc.) static, unless you properly manage them. 

https://github.com/libgdx/libgdx/wiki/Managing-your-assets

говорят что это может вызвать проблемы с приложением на андроиде. ибо там как то по хитрому работают статические переменные 
This typically would cause black/missing textures or incorrect assets.